src/js/sharedEth/connectedWalletPorts.js

```diff
@@ -158,7 +158,7 @@ function subscribeToTryConnect(app, eth, globEthereum, defaultNetworkId) {
 
       // We'll try to set to the user's last chosen provider, otherwise
       // defaulting to Web3.
-      let providerType = Number(storage('chosenProvider').get(PROVIDER_TYPE_WEB3));
+      let providerType = providerTypeId(globEthereum);
```

Comment:
Would recommend using the `chosenProvider` option first and then pass in the provider type id as the default value.

`let providerType = Number(storage('chosenProvider').get(providerTypeId(globEthereum)));`

Reply:
> Would recommend using the `chosenProvider` option first and then pass in the provider type id as the default value.
> 
> `let providerType = Number(storage('chosenProvider').get(providerTypeId(globEthereum)));`

Awesome! Appreciate your eyes on this. You know this codebase. Change made here https://github.com/compound-finance/compound-components/pull/67/commits/416209e58bf63eb29ec23900f8418aeffb4d31f0